### PR TITLE
Add gRPC e2e test

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: command/forwarder/root.go [hunk 1]
```diff
@@ -106,12 +106,19 @@ func Command() *cobra.Command {
...
+	// Add test commands.
+	test := &cobra.Command{
+		Use:   "test",
+		Short: "Run test servers for various protocols",
+	}
+	test.AddCommand(
+		grpc.Command(),
+		httpbin.Command(),
 	)
+	cmd.AddCommand(test)
```
Comment (Choraden) [inline command/forwarder/root.go:118]: nit: Test command could be moved to `/command/test/test.go`

Comment (mmatczuk) [reply command/forwarder/root.go:118]: It's not as grate as it sounds. The usefulness of such command is negligible.

Comment (Choraden) [reply command/forwarder/root.go:118]: ok

**Comment by Choraden** [comment]
LGTM

I wish we had better solution to configure test env: `t.Setenv("HTTPS_PROXY", proxy) // set proxy for grpc.Dial` and
`$(shell docker inspect forwarder-e2e-httpbin-1 --format='{{ range .Config.Env }}{{ if eq "FORWARDER" (slice . 0 9) }} -e "HTTPBIN{{slice . 9}}"{{end}}{{end}}' 2> /dev/null)`.

**Comment by mmatczuk** [comment]
I'm open to rewrite possibilities.

**Pull request merged**